components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Publisher/src/Services/PublisherJobService.cs

```diff
@@ -275,7 +275,8 @@ public sealed class PublisherJobService : IPublishServices<string> {
                             BatchTriggerInterval = DefaultBatchTriggerInterval.Value,
                             DiagnosticsInterval = TimeSpan.FromSeconds(60),
                             MaxMessageSize = 0,
-                            MaxOutgressMessages = DefaultMaxOutgressMessages.Value
+                            MaxOutgressMessages = DefaultMaxOutgressMessages.Value,
+                            HasRoutingInfo = false,
```

Comment:
Just going to put it out there, though I guess the answer is no since we do not plan to add any feature to the cloud microservices ... Should we add an env var for configuring this value on the cloud side ?

Reply:
you're right: the answer is no :). as an emergency people can use the cosmosdb directly to control the EnabeRoutingInfo.